src/Models/User/User.php

```diff
@@ -116,4 +120,22 @@ public function newsComments(): HasMany
     {
         return $this->hasMany(NewsComment::class);
     }
+
+    /**
+     * @return HasMany
+     */
+    public function questions(): HasMany
+    {
+        return $this->hasMany(Question::class, 'enquirer_id')
+            ->where('enquirer_id', $this->id);
+    }
+
+    /**
+     * @return HasMany
+     */
+    public function answers(): HasMany
```

Comment:
Hmm, would probably use `answeredQurestions()` here as it interferes less with potential "answers" to other stuff like comments?

Reply:
I'll make this more specific